lecture/models.py

```
@@ -1,26 +1,55 @@
+    title = models.CharField(max_length=255)
+    description = models.TextField(null=True, blank=True)
+    video_url = models.URLField(max_length=200, null=True, blank=True)
+    content = models.TextField(null=True, blank=True)
+    order = models.PositiveIntegerField(default=0)
+
     slug = models.SlugField(max_length=255, unique=True, blank=True)
 
+    class Meta:
+        ordering = ['order']
+
     def __str__(self):
```

Ruff (0.8.2)

Missing return type annotation for special method `__str__`

Add return type annotation: `str`

(ANN204)